doc/rst/language/spec/procedures.rst

```diff
@@ -1165,7 +1165,7 @@ listed in the following table:
 arity    operators
```

Comment:
I think you missed updating the `operator-name` syntax section at the top of the `Procedure Definitions` section

Reply:
thanks for the hint